api/src/main/java/org/pmiops/workbench/access/AccessSyncServiceImpl.java

```diff
@@ -36,7 +36,7 @@ public class AccessSyncServiceImpl implements AccessSyncService {
   private final AccessModuleService accessModuleService;
   private final InstitutionService institutionService;
   private final UserDao userDao;
-  private final InitialCreditsExpirationService initialCreditsExpirationServiceImpl;
+  private final FreeTierBillingService freeTierBillingService;
```

Comment:
(just here to make this comment - not doing a review)

I love the idea of combining these.  But in terms of naming, I'd prefer for the combined version to reference "Initial Credits" instead of "Free Tier".

Reply:
Totally agree!